### PR TITLE
Docs: add spanish readme translation

### LINKED ISSUE #2810: Add spanish language README
### Have you read the Documentation Contribution Guidelines?

- [X] I have read the [Documentation Contribution Guidelines](https://wails.io/community-guide#ways-of-contributing).

### Description

There is currently no README.es.md, it would be nice to have the translation of the README for spanish speaking countries.

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by k1nho**
# Description

This change introduces readme.es.md which translates the original README.md into the spanish language.

Fixes #2810 

## Type of change
  
Please delete options that are not relevant.

- [x] This change requires a documentation update

# Checklist:

- [ ] I have updated `website/src/pages/changelog.mdx` with details of this PR
- [ ] My code follows the general coding style of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    <img alt="GitHub tag (latest SemVer pre-release)" src="https://img.shields.io/github/v/tag/wailsapp/wails?include_prereleases&label=version"/>
+  </a>
+</p>
+
+<div align="center">
+<strong>
+<samp>
+
+[English](README.md) · [简体中文](README.zh-Hans.md) · [日本語](README.ja.md) ·
+[한국어](README.ko.md) · [Spanish](README.es.md)
```
Comment (leaanthony) [inline README.es.md:44]: Would it be better to use "Español" here?

File: README.md [hunk 2]
```diff
@@ -40,7 +40,8 @@
 <strong>
 <samp>
 
-[English](README.md) · [简体中文](README.zh-Hans.md) · [日本語](README.ja.md) · [한국어](README.ko.md)
+[English](README.md) · [简体中文](README.zh-Hans.md) · [日本語](README.ja.md) ·
+[한국어](README.ko.md) · [Spanish](README.es.md)
```
Comment (leaanthony) [inline README.md:44]: Same here :-)

**Comment by leaanthony** [comment]
Thanks for opening this. Just a small question 👍

File: README.es.md (hunk 1 above)
Comment (k1nho) [reply README.es.md]: Yep, this makes sense, thanks for the feedback! Also added the Español link to the other READMEs as well

**Review by leaanthony — APPROVED** [review]
Thanks for taking the time to do this 🙏

**Pull request merged**